c7n/resources/appelb.py

```diff
@@ -203,7 +272,20 @@ def initialize(self, albs):
 
 @filters.register('listener')
 class AppELBListenerFilter(ValueFilter, AppELBListenerFilterBase):
-    """
+    """Filter to pull ELBs with matching listeners
+
+    :example:
+
+        .. code-block: yaml
+
+            policies:
+              - name: appelb-https-listener
+                resource: app-elb
+                filters:
+                  - type: listener
+                    key: length([?Protocol=='HTTPS'])
```

Comment:
this expression is overly complex for the purpose or documentation, vs a simple key: Protocol value: HTTPS